source/client-side-encryption/tests/spawningMongocryptd.template.yml

```diff
@@ -0,0 +1,95 @@
+runOn:
```

Comment:
Unfortunately I think all these tests are suspect.  

1. The first one (Bypassing spawning mongocryptd fails to connect) fails when run non-in-isolation, because the previous test has already spawned mongocryptd and so this test is able to connect to the previously spawned one.
2. The second one (Spawning mongocryptd on a different port fails to connect) succeed for the wrong reason: all but the first spawned mongocrypd will fail to spawn (duplicate pid)
3. The third one (Insert on mongocryptd spawned with different URI) fails because the mongocryptd fails for the same reason as the second succeeds: the mongocryptd on port 27030 doesn't actually spawn, and so we fail to connect on that port.